src/js/view/rightclick.js

```diff
@@ -21,7 +21,7 @@ define([
         JW_CSS_NONE = 'none',
         JW_CSS_WHITE = '#FFF';
 
-    var Rightclick = function(_playerElement, _model) {
+    var Rightclick = function(_playerElement, _api, _model) {
```

Comment:
Can we instead store the current provider in the model and fetch from there
model.get('provider')?